### PR TITLE
Addition of flag for solr command and ascii test formatting dues to e…

### PR DESCRIPTION
**Pull request by jmwenda**
…rrors when mining fix

### CODE REVIEW

File: hypermap/aggregator/models.py [hunk 1]
```diff
@@ -356,9 +372,11 @@ def update_thumbnail(self):
             print 'Thumbnail updated for layer %s' % self.name
 
     def worldmap_date_miner(self):
-        year = re.search('\d{2,4} ?B?CE', str(self.title))
+        self.title = str(self.title.encode("ascii", "ignore"))
```
Comment (capooti) [inline hypermap/aggregator/models.py]: why are you encoding to ascii?

Comment (jmwenda) [inline hypermap/aggregator/models.py]: this is due to characters that are in other languages no english. i.e strings such as "Canada et Louisiane / par le Sr. le Rouge ingénieur géographe du Roy." or German terms with characters such as umlauts cause the re.search to bring up an error. Using unicode raised the following error for such terms. 
*** UnicodeEncodeError: 'ascii' codec can't encode character u'\xe9' in position 45: ordinal not in range(128)
Ascii format removes any of these special language characters. Utf-8 still has them. Please advise

**Comment by capooti** [comment]
@jmwenda I remember you were still working on this no?
the idea was to parse strings as utf8 and then in an exception block to parse it in ascii and then skip. Did you send another PR with this? I cannot find it

**Comment by jmwenda** [comment]
I did share this repo with you on  the changes requested before issuing a pull request.. https://github.com/cga-harvard/HHypermap/compare/master...jmwenda:datesfixsolr?expand=1

**Pull request closed without merging**